packages/nebula/nebula.0.2.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "1.2"
+maintainer: "Jesse Haber-Kucharsky <jesse@haberkucharsky.com>"
+authors: "Jesse Haber-Kucharsky <jesse@haberkucharsky.com>"
+homepage: "https://github.com/hakuch/Nebula"
+bug-reports: "https://github.com/hakuch/Nebula"
+license: "Apache 2.0"
+dev-repo: "git@github.com:hakuch/Nebula.git"
+build: [make]
+build-doc: [make "doc"]
+depends: [
+  "cmdliner" {= "0.9.7"}
```

Comment:
Would it be possible to make these minimum constraints and not exact ones?  We dont put upper bounds unless there is a known incompatibility.  Thanks!